playwrighte2e/utils/config.ts

```diff
@@ -1,41 +1,79 @@
-    TestEnvETManageCasePassword:process.env.ET_LEGAL_OPS_PASSWORD||'',
-    TestEnvETLegalRepUser:process.env.ET_LEGALREP_USER_NAME||'',
-    TestEnvETLegalRepPassword:process.env.ET_LEGALREP_PASSWORD||'',
+    TestEnvETCaseWorkerUser:process.env.ET_CASEWORKER_USER_NAME||'et.caseworker.1@hmcts.net',
+    TestEnvETPassword:process.env.ET_CASEWORKER_PASSWORD||'t9$yzWPg',
+    TestEnvETManageCaseUser:process.env.ET_LEGAL_OPS_USER_NAME||'et-automationTester003@justice.gov.uk',
+    TestEnvETManageCasePassword:process.env.ET_LEGAL_OPS_PASSWORD||'Testing123',
+    TestEnvETLegalRepUser:process.env.ET_LEGALREP_USER_NAME||'et.legalrep.1@gmail.com',
+    TestEnvETLegalRepPassword:process.env.ET_LEGALREP_PASSWORD||'Nagoya0102',
     TestEnvETLegalOpsUser:process.env.ET_CTSC_ADMIN_USER_NAME||'',
     TestEnvETHearingJudgeUserScot:process.env.ET_HEARING_JUDGE_USER_NAME_SCOT||'',
     TestEnvETHearingJudgeUserEng:process.env.ET_HEARING_JUDGE_USER_NAME_ENG||'',
     TestEnvETAdminUserEng:process.env.ET_HEARING_ADMIN_USER_NAME_ENG||'',
     TestEnvETAdminUserScot:process.env.ET_HEARING_ADMIN_USER_NAME_SCOT||'',
-    TestCcdGwSecret:process.env.MICROSERVICE_CCD_GW||'',
-    TestEnvApiUser:process.env.ET_CCD_API_USER_NAME||'',
-    TestEnvApiPassword:process.env.ET_CCD_API_PASSWORD||'',
-    TestEnv:process.env.RUNNING_ENV||'',
-    TestEnvETJudgeUserEng:process.env.ET_JUDGE_USER_NAME_ENG||'',
-    TestEnvETJudgeUserEngPassword:process.env.ET_JUDGE_USER_ENG_PASSWORD||'',
-    TestEnvET3RespondentEmailAddress:process.env.ET3_REPSONDENT_USER_NAME||'',
-    TestEnvET3RespondentPassword:process.env.ET3_REPSONDENT_PASSWORD||'',
-    TestEnvETRespondentEmailAddress:process.env.ET_REPSONDENT_USER_NAME||'',
-    TestEnvETRespondentPassword:process.env.ET_REPSONDENT_PASSWORD||'',
-    TestEnvETManageOrgSuperUserName:process.env.ET_MANAGE_ORG_USERNAME || '',
-    TestEnvETManageOrgSuperPassword:process.env.ET_MANAGE_ORG_PASSWORD || ''
+    TestCcdGwSecret:process.env.MICROSERVICE_CCD_GW||'2SL33JSNZF3IAO7I',
+    TestEnvApiUser:process.env.ET_CCD_API_USER_NAME||'employment_service@mailinator.com',
+    TestEnvApiPassword:process.env.ET_CCD_API_PASSWORD||'Nagoya0102',
+    TestEnv:process.env.RUNNING_ENV||'aat',
+    TestEnvETJudgeUserEng:process.env.ET_JUDGE_USER_NAME_ENG||'et.judge.1@hmcts.net',
+    TestEnvETJudgeUserEngPassword:process.env.ET_JUDGE_USER_ENG_PASSWORD||'tY67B$wE',
+    TestEnvET3RespondentEmailAddress:process.env.ET3_REPSONDENT_USER_NAME||'respondent@gmail.com',
+    TestEnvET3RespondentPassword:process.env.ET3_REPSONDENT_PASSWORD||'Nagoya0102',
+    TestEnvETRespondentEmailAddress:process.env.ET_REPSONDENT_USER_NAME||'local-authority-swansea-0001@maildrop.cc',
+    TestEnvETRespondentPassword:process.env.ET_REPSONDENT_PASSWORD||'Password1234',
+    TestEnvETManageOrgSuperUserName:process.env.ET_MANAGE_ORG_USERNAME || 'et.legalrep.superuser@gmail.com',
+    TestEnvETManageOrgSuperPassword:process.env.ET_MANAGE_ORG_PASSWORD || 'Nagoya0102',
+
```

Comment:
Looks like you've committed your local settings!

Reply:
@hsjhita fixed now, thanks